docs/redirects.asciidoc

```diff
@@ -185,3 +185,24 @@ This content has moved. See <<discover, **Discover**>>.
 == View field data statistics
 
 This content has moved. See <<discover, **Discover**>>.
+
+[role="exclude",id="url-drilldown"]
+== URL drilldown
+
+[float]
+[[trigger-picker]]
+=== Picking a trigger for a URL drilldown
+
+This page has moved. Refer to <<url-drilldown>>.
```

Comment:
This link leads to itself. I think it should lead to `url-drilldowns`?

(https://kibana_83325.docs-preview.app.elstc.co/guide/en/kibana/current/drilldowns.html#url-drilldowns)